dev/com.ibm.ws.jdbc/src/com/ibm/ws/jdbc/internal/JDBCDriverService.java

```diff
@@ -129,7 +129,7 @@
      * We should remove these classes from the Introspector cache
      * when the jdbcDriver is deactivated or modified.
      */
-    private final HashSet<Class<? extends CommonDataSource>> introspectedClasses = new HashSet<Class<? extends CommonDataSource>>();
```

Comment:
We do not introspect the Driver impl.  This ought to remain typed as CommonDataSource.

Reply:
Updated